### ISSUE TITLE
0.18.0 is missing hljs css

### ISSUE THREAD

**Issue by laat** [asyncapi/html-template#157]
#### Describe the bug

the html is missing hljs styles

#### How to Reproduce

```sh
npx @asyncapi/generator --force-write --output test \
  https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml \
  @asyncapi/html-template
```

The HTML is missing syntax highlighting in the examples

![image](https://user-images.githubusercontent.com/146559/108514776-8fffe200-72c4-11eb-8da1-ff9293c4714e.png)

### 0.17.1 worked

this is a regression from 0.17.1, where syntax highlighting worked

```sh
npx @asyncapi/generator --force-write --output test \
  https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml \
  @asyncapi/html-template@0.17.1
```

![image](https://user-images.githubusercontent.com/146559/108515914-ea4d7280-72c5-11eb-9565-a90a4a63e662.png)

**Comment by Theiaz** [comment]
Possible fix could be #159.

**Comment by derberg** [comment]
@laat it will be fixed as @Theiaz once we merge the fix, I checked and with the PR css is fixed:

```
npx @asyncapi/generator --force-write --output test   https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml https://github.com/Theiaz/html-template#158-use-css-imports
```
